client/packages/system/src/Stock/Components/Repack/RepackModal.tsx

```diff
@@ -122,15 +122,16 @@ export const RepackModal: FC<RepackModalControlProps> = ({
       };
     }
 
-    const { numberOfPacks, packSize } = repackData.from;
+    const { numberOfPacks, packSize, location: oldLocation } = repackData.from;
     const { packSize: newPackSize, location } = repackData.to;
     return {
       stockLineId: stockLine.id,
       numberOfPacks,
       packSize,
       newPackSize,
-      locationName: location?.name,
+      locationName: oldLocation ? oldLocation.name : location?.name,
```

Comment:
this would show old location as new location even if old location was not set, Would it be ok to just map from.location to locationName and new to.location to newLocationName ?